src/getEntityTypescriptPostgres.ts

```diff
@@ -379,7 +376,7 @@ const generateEntityString = (
 		`import { Entity, ${
 			isContainsForeignKeys || enumsKeys.length > 0 ? "Field, " : ""
 		}Fields } from 'remult'` +
-		`${addLineIfNeeded([...new Set(additionnalImports)], (c) => `${c}`)}` +
+		`${addLineIfNeeded([...new Set(additionnalImports)], (c) => c)}` +
```

Comment:
maybe `addLineIfNeeded` can get an optional format function

Reply:
It can. Is it game changer?

Reply:
No you dont have to do it now it NTH